connectors/src/connectors/confluence/lib/confluence_client.ts

```diff
@@ -31,6 +31,14 @@ const ConfluenceListSpacesCodec = t.type({
   results: t.array(ConfluenceSpaceCodec),
 });
 
+const ConfluencePaginatedResults = <C extends t.Mixed>(codec: C) =>
+  t.type({
+    results: t.array(codec),
+    _links: t.partial({
```

Comment:
Why the `t.partial` here VS `next: t.union([t.string, t.undefined])` ?

Reply:
`t.partial` means that the key does not need to exist vs `t.undefined` that explicitly checks for the key to exist and have the value `undefined`.

Reply:
TIL ! I'm surprised this is being treated differently by io-ts !